README.md

```diff
@@ -1 +1,20 @@
 # hw01
+
+
+# About Me
+My name is **Anastasiya Apalkova** and I am a student in the MAPSS program concentrating in psychology. 
+I was born in Ukraine and moved to Chicago with my family when I was seven years old. 
+***
+![Ukraine Map](https://res.cloudinary.com/fen-learning/image/upload/c_limit,w_505,h_490/infopls_images/images/mukraine.gif) ![picture of me](pic4.jpg)
+***
+I did my undergrad at [UW-Madison](https://www.wisc.edu/) where I majored in psychology and sociology. 
+During my MAPSS year I will be completing a _thesis_ in social psychology as well as working on improving my 
+_computational and quantitative skills_ to prepare for PhD study. 
+During my free time I enjoy:
+
+* Reading 
```

Comment:
Nice use of the list as a way to organize and make the information easier to read!